orion/pkg/repos/repos_util.go

```diff
@@ -27,3 +28,16 @@ func handleSqlExecResult(result sql.Result, expected int64, errorMessage string)
 	}
 	return nil
 }
+
+func appendError(errorString string, stringId string, rowId uint, err error) string {
+	if len(stringId) > 0 {
+		errorString += stringId + ": "
+	}
+	if rowId > 0 {
+		errorString += fmt.Sprint(rowId) + ": "
+	}
+	if err != nil {
+		errorString += err.Error() + "\n"
```

Comment:
It would be great if in the error message, you made it more complete. Remember, you're using error messages to help you debug erros in the future, so it's good to be clear what's happening.

I would do:
```
"id: " + id + " error: " + err.Error() + ", "
```
Or something like that. So when you actually see the error message you know what values are being mapped to what keys.